<ui version="4.0" >
 <class>QmitkVolumetryViewControls</class>
 <widget class="QWidget" name="QmitkVolumetryViewControls" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QmitkDataStorageComboBox" name="m_ImageSelector" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" name="_2" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_CalcButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>C&amp;alc!</string>
         </property>
         <property name="shortcut" >
          <string>Alt+A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_TimeSeriesButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Time series</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>149</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_SaveCsvButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Save CSV</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_3" >
       <item>
        <layout class="QHBoxLayout" name="_4" >
         <item>
          <widget class="QSlider" name="m_ThresholdSlider" >
           <property name="maximum" >
            <number>1000</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_ThresholdLineEdit" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>50</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text" >
            <string>1000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="m_Result" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="m_TextEdit" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer1" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >QmitkDataStorageComboBox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
